data/abilities.js

```diff
@@ -735,6 +735,9 @@ exports.BattleAbilities = {
 				return null;
 			}
 		},
+		onEnd: function (pokemon) {
+			pokemon.removeVolatile('flashfire');
```

Comment:
I'm not sure if `onEnd` is the best way to do it?

Aren't there ways to remove an ability without running `onEnd`?